docs/.vuepress/config.js

```diff
@@ -75,7 +75,7 @@ var DOCS_VERSION = '${getThisDocsVersion()}';
   markdown: {
     toc: {
       includeLevel: [2, 3],
-      containerHeaderHtml: '<div class="toc-container-header">Table of contents</div>'
+      containerHeaderHtml: '<div class="toc-container-header">In this article</div>'
```

Comment:
```suggestion
      containerHeaderHtml: '<div class="toc-container-header">In this article:</div>'
```

Reply:
@kirszenbaum, I would leave it as it is. It's a header, not an unordered sort of list.

MS presents the similar approach: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements#global-namespace